swagger/api.go

```diff
@@ -416,5 +417,34 @@ func (a *API) Walk(callback func(path string, endpoints *Endpoint)) {
 
 // RenderJSON returns the static swagger schema, as json
 func (a *API) RenderJSON() ([]byte, error) {
+	if !a.GeneratePrivate {
```

Comment:
Do we need this parameter? You could just write `someAPI.RemovePrivate(); someAPI.RenderJSON()`, or if `RemovePrivate` is changed to return `a` at the end, you could just write `someAPI.RemovePrivate().RenderJSON()`.